script/test/cmd/tests.sh

```diff
@@ -394,6 +394,15 @@ cd $CURRENT_DIR
 
 # Test V3 Support of Docker Compose
 
+# Test deploy mode: global
+cmd="kompose convert --stdout -j -f $KOMPOSE_ROOT/script/test/fixtures/v3/docker-compose-deploy-mode.yaml"
+sed -e "s;%VERSION%;$version;g" -e "s;%CMD%;$cmd;g"  "$KOMPOSE_ROOT/script/test/fixtures/v3/output-deploy-mode-k8s-template.json" > /tmp/output-k8s.json
+convert::expect_success "kompose convert --stdout -j -f $KOMPOSE_ROOT/script/test/fixtures/v3/docker-compose-deploy-mode.yaml" "/tmp/output-k8s.json"
```

Comment:
use `convert::expect_success_and_warning` to also check warning to avoid false positives with the wrong docker compose file